### PR TITLE
build: update deps to run on node 20

### PR DESCRIPTION
**Pull request by vmaerten**

### CODE REVIEW

File: .github/workflows/deployGHPages.yml [hunk 1]
```diff
@@ -13,10 +13,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
+    - name: Use Node.js
       uses: actions/setup-node@v2
       with:
-        node-version: 16.14.2
+        node-version: 20.x
```
Comment (bbbenja) [inline .github/workflows/deployGHPages.yml:19]: I preferred fixed version, I recently have an issue on a build with the new `18.18`. Maybe `20.7.x` is good enough.

File: .github/workflows/firebase-hosting-pull-request.yml [hunk 2]
```diff
@@ -8,10 +8,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Use Node.js 16.x
+      - name: Use Node.js 20.x
         uses: actions/setup-node@v2
         with:
-          node-version: 16.14.2
+          node-version: 20.x
```
Comment (bbbenja) [inline .github/workflows/firebase-hosting-pull-request.yml:14]: same here

**Comment by bbbenja** [comment]
Photos dont work, is it an API key restriction ?

**Comment by vmaerten** [comment]
> Photos dont work, is it an API key restriction ?

Fixed !

**Review by bbbenja — APPROVED** [review]

**Pull request merged**